db/seeds.rb

```diff
@@ -4,8 +4,7 @@
 user = User.find_or_initialize_by(:email => ENV['SEED_EMAIL'] || "admin@example.com")
 
 if user.persisted?
-  puts "User with email '#{user.email}' already exists, not seeding."
-  exit
+  raise "User with email '#{user.email}' already exists, not seeding."
```

Comment:
Good point, would you like to add specs for the seeds in this PR or is it a different scope?